Library/Homebrew/keg_relocate.rb

```diff
@@ -186,6 +225,23 @@ def each_unique_file_matching(string)
     end
   end
 
+  def each_unique_binary_file(string)
```

Comment:
```suggestion
  def each_unique_binary_file_matching(string)
```
I think this makes method names more consistent, no?

Reply:
Ha, when I copied these changes back from my testing repo I bungled them.  There aren't even supposed to be any arguments to this method, because the string `grep` searches for is fixed to be "\x00", not the prefix.  I'll push the correct version of this and we can see if still make sense to change the name!